### PR TITLE
fix: Added assets, including missing favicons

### PR DESCRIPTION
**Pull request by ojeytonwilliams**
- [x] I have read [freeCodeCamp's contribution guidelines](https://github.com/freeCodeCamp/freeCodeCamp/blob/master/CONTRIBUTING.md).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `master` branch of freeCodeCamp.
- [x] None of my changes are plagiarized from another source without proper attribution.
- [x] All the files I changed are in the same world language (for example: only English changes, or only Chinese changes, etc.)
- [x] My changes do not use shortened URLs or affiliate links.

This is a stab at fixing #35219.

The static/assets directory existed in the old version of the client and contained the missing favicons.  This PR simply restores that.  Some of these assets are pulled in by https://github.com/freeCodeCamp/freeCodeCamp/blob/43a0705a913819406000c5c1d63dbd359b18cc8c/client/src/head/favicons.js, but not all of them.  I assumed that all of them should be, but I wasn't sure.  If anyone knows better, let me know and I'll change things appropriately.

Sorry if this is a little less than perfect, but I figured I shouldn't delay given that this is on the critical path.

### CODE REVIEW

File: client/static/assets/site.webmanifest [hunk 1]
```diff
@@ -0,0 +1,54 @@
+{
+    "name": "Learn | freeCodeCamp",
```
Comment (raisedadead) [inline client/static/assets/site.webmanifest]:
```suggestion
    "name": "freeCodeCamp.org",
```

File: client/static/assets/site.webmanifest [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+    "short_name": "Learn | freeCodeCamp",
```
Comment (raisedadead) [inline client/static/assets/site.webmanifest]:
```suggestion
    "short_name": "fCC",
```

**Review by raisedadead — CHANGES_REQUESTED** [review]
A few minor changes. Could we use the existing plugin, or a better plugin and have it handled in gatsby to generate the manifest?

**Comment by ojeytonwilliams** [comment]
@raisedadead So, it turns out the Gatsby plugin is already generating a manifest and some icons.  I had thought the problem was due to a lack of assets, but it was just that trying and failing to find these assets confuses Firefox.  If they're never looked for, it is fine.

If you're happy with what that plugin produces, we can simply remove the redundant icons and manifest.  I'll update this PR so you can see that in action.

**Review by raisedadead — APPROVED** [review]

**Comment by raisedadead** [comment]
Going to re-review the approach here.

**Pull request merged**